.github/workflows/scala.yml

```diff
@@ -109,6 +109,11 @@ jobs:
         run: |
           sleep 1
           sbt --no-colors compile
+      - name: Build Base Java Extensions
+        shell: bash
+        run: |
+          cd std-bits
+          mvn package
```

Comment:
If I understand correctly, the tests will not run unless `mvn package` has been run.

I think it should at least be added to the `CONTRIBUTING` doc (and also `mvn` probably needs to be added under `System Requirements` there, so that users know that they need to install it).

Ideally, calling `mvn package` should be handled by sbt automatically whenever tests are run (or if `package` is not incremental and so re-runs on every call, only when changes were made to `std-bits`) in a similar way that the `buildNativeImage` task is called before running tests if the launcher code was modified.

Reply:
OK, done